lib/suspenders/actions.rb

```diff
@@ -31,6 +31,28 @@ def expand_json(file, data)
       action ExpandJson.new(destination_root, file, data)
     end
 
+    def gem(*args)
+      options = args.extract_options!
+      name, *versions = args
+
+      parts, message = [quote(name)], name.dup
+
+      if (versions = versions.any? ? versions : options.delete(:version))
+        versions = Array(versions)
+        versions.each do |version|
+          parts << quote(version)
+        end
+        message << " (#{versions.join(", ")})"
+      end
+
+      parts << quote(options) unless options.empty?
+
+      str = []
+      str << indentation
```

Comment:
Not a big deal, but I prefer how it reads in the original method:

```ruby
    str = "gem #{parts.join(", ")}"
    str = indentation + str
```

Also, the original method uses `in_root,` but I suppose that is not necessary here.

Reply:
Oddly enough, I was getting failures when I used that implementation, but I'm not totally sure why 🤔

I think this might have also been the result of having run `standardrb --fix`.